diff --git a/src/Bridge/Elasticsearch/DataProvider/CollectionDataProvider.php b/src/Bridge/Elasticsearch/DataProvider/CollectionDataProvider.php
index b69f0083a..e410c10bd 100644
--- a/src/Bridge/Elasticsearch/DataProvider/CollectionDataProvider.php
+++ b/src/Bridge/Elasticsearch/DataProvider/CollectionDataProvider.php
@@ -64,7 +64,7 @@ final class CollectionDataProvider implements ContextAwareCollectionDataProvider
     {
         try {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            if (false === $resourceMetadata->getCollectionOperationAttribute($operationName, 'elasticsearch', true, true)) {
+            if (false === $resourceMetadata->getCollectionOperationAttribute($operationName, 'elasticsearch', false, true)) {
                 return false;
             }
         } catch (ResourceClassNotFoundException $e) {
diff --git a/src/Bridge/Elasticsearch/DataProvider/ItemDataProvider.php b/src/Bridge/Elasticsearch/DataProvider/ItemDataProvider.php
index 9d3fcb1c7..4cce1b9a2 100644
--- a/src/Bridge/Elasticsearch/DataProvider/ItemDataProvider.php
+++ b/src/Bridge/Elasticsearch/DataProvider/ItemDataProvider.php
@@ -58,7 +58,7 @@ final class ItemDataProvider implements ItemDataProviderInterface, RestrictedDat
     {
         try {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            if (false === $resourceMetadata->getItemOperationAttribute($operationName, 'elasticsearch', true, true)) {
+            if (false === $resourceMetadata->getItemOperationAttribute($operationName, 'elasticsearch', false, true)) {
                 return false;
             }
         } catch (ResourceClassNotFoundException $e) {
